src/ORM/Behavior/Translate/TranslateTrait.php

```diff
@@ -46,8 +46,10 @@ public function translation($language)
             $created = true;
         }
 
-        if ($created || empty($i18n[$language]) || !($i18n[$language] instanceof Entity)) {
-            $i18n[$language] = new Entity();
+        if ($created || empty($i18n[$language]) || !($i18n[$language] instanceof EntityInterface)) {
+            $className = get_class($this);
+
+            $i18n[$language] = new $className;
```

Comment:
Well, doesn't mean we have a sniff for it, but usually all class inits have the trailing `()`.